module/block_iterator/state.go

```diff
@@ -55,8 +55,11 @@ func (n *PersistentIteratorState) NextRange() (rg module.IteratorRange, hasNext
 		return module.IteratorRange{}, false, nil
 	}
 
+	// latest could be less than next, if the pruning-threshold was increased, which means
```

Comment:
Is this something that could be info or warn logged?

Reply:
This comment makes it seem like this is a pruning related code branch in a component that is supposed to be more general. Is this ok?

Reply:
[Updated in this comments](https://github.com/onflow/flow-go/pull/7112/commits/33a24ace59ae58378b5d1076c27c6b1bdaffddf6)